andela/mrm_api
# #162953051: Allow missing items to be provided on room feedback

#### What does this PR do?
- Allow Users to add missing items to a room feedback

#### How should this be manually tested?
- Run the server
- Test the query at `localhost:5000/mrm`
- Run `createResponse` mutation with the parameters `room_id, question_id(for check question type)`

#### What are the relevant pivotal tracker stories?
[#162953051](https://www.pivotaltracker.com/story/show/162953051)

#### Screenshots
1. Response for Non-existence roomID
<img width="1219" alt="screen shot 2019-01-10 at 11 05 41 am" src="https://user-images.githubusercontent.com/23406358/50961428-0451df80-14c8-11e9-92c6-55ea73f51818.png">


2. Response for non-existence resource in the room
<img width="1215" alt="screen shot 2019-01-10 at 11 06 17 am" src="https://user-images.githubusercontent.com/23406358/50961467-1764af80-14c8-11e9-9646-d561881215d7.png">


3. Response for Missing item parameter omitted
<img width="1219" alt="screen shot 2019-01-10 at 11 07 05 am" src="https://user-images.githubusercontent.com/23406358/50961509-264b6200-14c8-11e9-9c01-bf711dc74b56.png">


4. Response for an empty list of missing items
<img width="1220" alt="screen shot 2019-01-10 at 11 06 49 am" src="https://user-images.githubusercontent.com/23406358/50961538-39f6c880-14c8-11e9-81d6-0daa5c95b74e.png">


5. Response for reported items
<img width="1214" alt="screen shot 2019-01-10 at 11 04 01 am" src="https://user-images.githubusercontent.com/23406358/50961557-47ac4e00-14c8-11e9-9dac-c2b4b125c35e.png">
